openvdb/openvdb/math/Coord.h

```diff
@@ -13,8 +13,7 @@
 #include "Math.h"
 #include "Vec3.h"
 
-namespace tbb { class split; } // forward declaration
-
+#include <tbb/blocked_range.h> // for tbb::split
```

Comment:
**Significant change**

I couldn't find a nice way to fix the forward declaration between versions.